bot/exts/evergreen/fun.py

```diff
@@ -71,7 +71,7 @@ async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         await ctx.send(output)
 
     @commands.command(name="uwu", aliases=("uwuwize", "uwuify",))
-    async def uwu_command(self, ctx: Context, *, text: str) -> None:
+    async def uwu_command(self, ctx: Context, *, text: clean_content(fix_channel_mentions=True)) -> None:
```

Comment:
If the command argument is a message ID or message jumplink, the command will internally fetch the message text on this line in the command:

```py
        text, embed = await Fun._get_text_and_embed(ctx, text)
```

This bypasses the `clean_content` converter (which is only applied to the ID/jumplink, not the message text that is fetched from it later).

Reply:
Good point, thanks. I'll get that fixed up.

Reply:
OK, I should have resolved this now, along with also applying the cleaning to the mixed case command so mentions don't scramble the raw mention format, instead always using the resolved names.